### ISSUE TITLE
Enhancement: update to 1.10.2

### ISSUE THREAD

**Issue by ArcusMG** [P3pp3rF1y/AncientWarfare2#508]
I absolutely love this mod and would love to some day see it updated to 1.10.2. Been enjoying a lot of the new and existing mods that have moved to 1.10.2 and would really like to add this to them. I understand if this is not something that is ever planned but would love to see it if it's at all possible.

**Comment by cosmicdan** [comment]
I am interested to do this (I'm working on new features via a fork), but I don't see it happening any time soon.

The mod is huge, and - no offense to ShadowMage at all, his work is fantastic - the code is pretty messy. Not the worst by far, but he certainly paid little consideration to accommodate other contributors or the future. It's been a struggle to understand why or how he did a lot of things the way he did.

With that said, if or when I do it, I would start a brand new mod from scratch - using AW2 as a base. Though I don't care about the vehicles or RF stuff personally, mostly just the NPC's - so it wouldn't be the same mod.

Judging by GotoLink's availability since taking over maintenance, I would be extremely surprised if he said he planned to update it.

In short: Free time from real life is a factor here, but working on AW2 is right up the top of my list. No idea about GotoLink himself, but I doubt he has any interest in doing so (and we shouldn't expect/blame him - nobody else of any significant modding skill even wanted to maintain the mod, so we're lucky as it is).

**Comment by ArcusMG** [comment]
Sounds great! These are free mods after all. I'm grateful for any work people decide to do. If you are able to do something new, I look forward to it.

The NPCs are by far my favorite part. Something really satisfying about making a new town and populating it with working NPCs who actually do stuff for you.

**Comment by Lohardt** [comment]
I am only using this mod for the NPC's/buildings and the world generation. I would love to see them in a 1.10.2 mod.

**Comment by wadewegner** [comment]
I'm in. Who's started this? I don't see a 1.10.2 branch.

**Comment by cosmicdan** [comment]
Nobody has started. I said that maybe I will one day, I would like to but it will be a re-write and not actually an update. GotoLink is the official maintainer of AW2, but he and I are not directly collaborating or anything.

**Comment by P3pp3rF1y** [comment]
Closing this, 1.10 isn't going to happen, but 1.12 is a thing.
